zk/stages/stage_sequence_execute_test.go

```diff
@@ -145,7 +146,7 @@ func TestSpawnSequencingStage(t *testing.T) {
 	zkCfg := &ethconfig.Zk{
 		SequencerResequence: false,
 		// lower batch close time ensures only 1 block will be created on 1 turn, as the test expects
-		SequencerBatchSealTime:      3 * time.Millisecond, // normally it is greater that block seal time, allows one more block to be added to the batch
+		SequencerBatchSealTime:      2 * time.Millisecond, // normally it is greater that block seal time, allows one more block to be added to the batch
```

Comment:
curious why. we can set it to anything ofc, I set it to the most meaningful case we have in prod - batch timeout > block timeout.